internal/cmd/backup.go

```diff
@@ -48,7 +71,53 @@ func createBackupFile(filename string) (*os.File, error) {
 	return f, nil
 }
 
-func backupCmdFunc(cmd *cobra.Command, args []string) error {
+var (
+	missingAllowedTypes = regexp.MustCompile(`(\s*)(relation)(.+)(\/\* missing allowed types \*\/)(.*)`)
```

Comment:
What stops this from getting greedy and gobbling up more than we anticipated?

Reply:
Not much other than the low odds of someone having exactly this match. I could hide this behind a flag and have it default off.

Reply:
✅ put this behind a flag so only folks that need it will use it